project-templates/migrations/move-child_name-to-patient_name/ddoc/index.js

```diff
@@ -0,0 +1,28 @@
+
+const path = require('path');
+
+require('dotenv').config({
+  path: path.resolve(__dirname, '..', 'env')
+});
+
+if (!process.env.MIGRATION_NAME) {
+  throw new Error("Missing environment variable: MIGRATION_NAME");
+}
+
+const NAME = process.env.MIGRATION_NAME;
+
+/*
+ * The CouchDB js interpreter by default is pretty old so it does not interpret
+ * ES5/6, so anything inside the ddoc object should be code that works on
+ * netscape navigator.
```

Comment:
AFAIK Couch 1.6.1 users Spidermonkey 1.8.5, which is ES5 compatible.

– https://github.com/apache/couchdb/blob/1.6.1/INSTALL.Unix#L45
– https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Releases/1.8.5#New_JavaScript_language_features